lib/administrate/search.rb

```diff
@@ -19,9 +19,10 @@ def run
     private
 
     delegate :resource_class, to: :resolver
+    delegate :table_name, to: :resource_class
 
     def query
-      search_attributes.map { |attr| "lower(#{attr}) LIKE ?" }.join(" OR ")
+      search_attributes.map { |attr| "lower(#{table_name}.#{attr}) LIKE ?" }.join(" OR ")
```

Comment:
@klaseskilson someone just opened a PR showing that this code doesn't align with the SQL standard in #829. Are you able to quote the table name and, just to leave things in better shape than they were before, the column name as well?